scoper.inc.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+return [
+    'whitelist' => [
+        'Psy\Shell',
+        'Psy\bin',
```

Comment:
if this means "things that shouldn't be renamed", then `Psy\sh`, `Psy\debug`, and `Psy\info` belong in this list as well.